### PR TITLE
Feature/99 style property and mlt fields

### PR DESCRIPTION
**Pull request by pixeljunkie**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | yes
| Tests pass?   | yes
| Fixed issues  | chameleon-system/chameleon-system#99
| License       | MIT

This branch is based on: https://github.com/chameleon-system/chameleon-base/pull/168

### CODE REVIEW

File: UPGRADE-6.3.md [hunk 1]
```diff
@@ -379,6 +379,7 @@ You should remove all width/height settings in CreateModalXY calls if it does no
 - loadStandaloneDocumentManager
 - PublishViaAjaxCallback()
 - src/CoreBundle/Resources/public/javascript/mainNav.js
+- showMLTField() - use CHAMELEON.CORE.MTTableEditor.switchMultiSelectListState(iFrameId, url) instead.
```
Comment (umulmrum) [inline UPGRADE-6.3.md]: Our best friend, the alphabetic order .. :-) (one line up, please)

File: src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldPropertyTable.class.php [hunk 2]
```diff
@@ -60,18 +60,30 @@ public function GetHTML()
...
-        </div>
-        <div id="'.TGlobal::OutHTML($this->name).'_iframe_block" class="listBoxBorder"><iframe id="'.TGlobal::OutHTML($this->name).'_iframe" width="100%" height="470" frameborder="0" style="display: none;"></iframe></div>';
+
+            $sEscapedName = TGlobal::OutHTML($this->name);
+
+            $html .= '<input type="hidden" id="' . TGlobal::OutHTML($this->name) . '" name="' . TGlobal::OutHTML($this->name) . '" value="' . TGlobal::OutHTML($sPropertyTableName) . '" />';
+            $html .= '<div class="card">
+            <div class="card-header p-1">
+                <div class="card-action" 
+                data-fieldstate="' . $stateContainer->getState($this->sTableName, $this->name) . '"
```
Comment (umulmrum) [inline src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldPropertyTable.class.php]: This value should be quoted.

File: src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldPropertyTable.class.php [hunk 3]
```diff
@@ -60,18 +60,30 @@ public function GetHTML()
...
-          <div class="listBoxTopTitleLeft"><img src="'.URL_CMS.'/images/boxTitleBgLeft.gif" alt="" width="4" height="22" border="0" hspace="0" vspace="0" /></div>
-          <div class="listBoxTop" data-fieldstate="'.$stateContainer->getState($this->sTableName, $this->name)."\" onClick=\"setTableEditorListFieldState(this, '{$sStateURL}');{$sOnClickEvent}\"><img src=\"".URL_CMS.'/images/icon_show_hide_list.gif" border="0" hspace="0" vspace="0" align="left" style="padding-top: 3px; padding-right: 8px; padding-left: 4px; ">'.TGlobal::Translate('chameleon_system_core.field_property.open_or_close_list').'</div>
-          <div class="cleardiv">&nbsp;</div>
-        </div>
-        <div id="'.TGlobal::OutHTML($this->name).'_iframe_block" class="listBoxBorder"><iframe id="'.TGlobal::OutHTML($this->name).'_iframe" width="100%" height="470" frameborder="0" style="display: none;"></iframe></div>';
+
+            $sEscapedName = TGlobal::OutHTML($this->name);
+
+            $html .= '<input type="hidden" id="' . TGlobal::OutHTML($this->name) . '" name="' . TGlobal::OutHTML($this->name) . '" value="' . TGlobal::OutHTML($sPropertyTableName) . '" />';
```
Comment (umulmrum) [inline src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldPropertyTable.class.php]: Concatenation dots aren't surrounded by space - see https://symfony.com/doc/current/contributing/code/standards.html

Could you run PHP-CS-Fixer again on changed code?

File: UPGRADE-6.3.md (hunk 1 above)
Comment (pixeljunkie) [reply UPGRADE-6.3.md]: done

File: src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldPropertyTable.class.php (hunk 2 above)
Comment (pixeljunkie) [reply src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldPropertyTable.class.php]: done

File: src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldPropertyTable.class.php (hunk 3 above)
Comment (pixeljunkie) [reply src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldPropertyTable.class.php]: done

**Comment by umulmrum** [comment]
Thanks @pixeljunkie

**Pull request merged**